pages/glossary/explicit-role.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Explicit Semantic Role
+key: explicit-role
+---
+
+The explicit semantic role is the [semantic role](#semantic-role) of an element set by its [role attribute](https://www.w3.org/TR/html/dom.html#aria-role-attribute).

```

Comment:
It's not linking to correct place. Was this what you intended? 
https://html.spec.whatwg.org/multipage/infrastructure.html#attr-aria-role

Reply:
This was indeed not the correct link. The one you suggest is just a redirection to WAI-ARIA. Digging into it, I do not find a precise definition of the `role` *attribute* (rather than the *concept* of roles), the best I have is https://www.w3.org/TR/wai-aria-1.2/#introroles which is itself mostly a redirection to https://www.w3.org/TR/role-attribute/

I guess I'll go for https://www.w3.org/TR/role-attribute/ then…